dev/com.ibm.ws.jdbc_fat_db2/publish/servers/com.ibm.ws.jdbc.fat.db2/server.xml

```diff
@@ -32,10 +33,14 @@
     <fileset dir="${shared.resource.dir}/db2"/>
   </library>
   
+  <library id="DB2LibJar">
+    <file name="${shared.resource.dir}/db2/jcc-11.1.4.4.jar"/>
```

Comment:
Two comments: 
- Having the version number on the jar makes it more difficult to service our test suites when upgrading drivers.  Instead, can we add a rename to the build.gradle file to remain the file to `jcc.jar`
```groovy
task copyDB2(type: Copy) {
  shouldRunAfter jar
  from configurations.db2
  into new File(autoFvtDir, 'publish/shared/resources/db2')
  rename 'jcc.*.jar', 'jcc.jar'
}
```
- Instead of creating a new library object here, can we instead modify DB2Lib with `includes="jcc.jar"`?